apps/explorer/src/api/assets/assetsApi.ts

```diff
@@ -16,27 +16,71 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { TokenList } from '@alephium/token-list'
-import { hexToString } from '@alephium/web3'
+import { NFTCollectionUriMetaData, NFTTokenUriMetaData } from '@alephium/shared'
+import { TOKENS_QUERY_LIMIT } from '@alephium/shared'
```

Comment:
These 2 lines can be merged. Let's start using this https://eslint.org/docs/latest/rules/no-duplicate-imports